pkg/installation/installation.go

```diff
@@ -217,7 +229,7 @@ func (i *Installation) validateConfigurations() error {
 		i.Options.remoteImage = i.Options.Source
 		i.Options.configVersion = "master"
 	default:
-		return fmt.Errorf("Failed to parse the source flag. It can take one of the following: 'local', 'latest', release version (e.g. 1.4.1), or installer image")
+		return fmt.Errorf("failed to parse the source flag. It can take one of the following: 'local', 'latest', release version (e.g. 1.4.1), or installer image")
```

Comment:
```suggestion
		return fmt.Errorf("failed to parse the source flag. It can take one of the following: 'local', 'latest', 'latest-published', release version (e.g. 1.4.1), or installer image")
```